src/Interactive/Host/Microsoft.CodeAnalysis.InteractiveHost.csproj

```diff
@@ -68,6 +68,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="StreamJsonRpc" />
+    <PackageReference Include="System.Threading.Tasks.Dataflow" />
```

Comment:
Let's add a comment - otherwise one might think this is unnecessary reference. I'm guessing this is a limitation of our (Roslyn's) binding redirect generation logic.

Reply:
Alright, done